tools/test/examples/examples.json

```diff
@@ -475,6 +475,22 @@
         "baud_rate": 9600,
         "compare_log": ["mbed-os-example-atecc608a/tests/atecc608a.log"],
         "auto-update" : true
+    },
+    {
+        "name": "mbed-bootloader",
+        "github":"https://github.com/ARMmbed/mbed-bootloader",
+        "sub-repo-example": false,
+        "subs": [],
+        "features" : [],
+        "targets" : ["K64F", "DISCO_L475VG_IOT01A", "NRF52840_DK", "NUCLEO_F303RE", "NUCLEO_F429ZI"],
+        "toolchains" : ["GCC_ARM"],
+        "exporters": [],
+        "compile" : true,
+        "export": false,
+        "test" : false,
+        "baud_rate": 115200,
+        "compare_log": ["mbed-bootloader/TESTS/booting.log"],
+        "auto-update" : true
```

Comment:
maybe here could just be some new field:

"custom_profile": "tiny.json"

which the CI would then use instead of the defaults?